src/Quarrel/Quarrel.csproj

```diff
@@ -482,11 +488,21 @@
     <Content Include="Assets\IconAssets\WideTile.scale-400.png" />
     <Content Include="Controls\Markdown\About.txt" />
     <Content Include="Controls\Markdown\oltheme.txt" />
+    <AdditionalFiles Include="appsettings.json" />
     <None Include="Package.StoreAssociation.xml" />
     <Content Include="Properties\Default.rd.xml" />
-    <Content Include="SodiumC.dll" />
+    <None Include="Quarrel_TemporaryKey.pfx" />
+  </ItemGroup>
```

Comment:
Temporary key again.

Reply:
Yeah, oversight.  You can remove that line.